### ISSUE TITLE
PY3: large vertical panel above the browser

### ISSUE THREAD

**Issue by declension** [quodlibet/quodlibet#2174]
Possibly just a local thing, but I'm consistently seeing this when running latest Git `master` under Python 3(.5.2), Gtk 3.20:

![image](https://cloud.githubusercontent.com/assets/3322808/21508367/88556058-cc78-11e6-8b56-43f4d493ae34.png)

**Comment by lazka** [comment]
This is py3 only?

**Comment by declension** [comment]
It seems to be, but let me check again running on PY2 to confirm

**Comment by declension** [comment]
No, my bad, if I force it to use py2 (IDE doesn't seem to respect the setting) this also happens. Updating title

**Comment by lazka** [comment]
I can't reproduce using 16.10 + Adwaita theme. Maybe a plugin? There is an issue about the lyrics plugin confusing gtk+

**Comment by declension** [comment]
Bingo! Having tried on this machine (PY2, 16.10, HiDPI) I've noticed it's fine until I enabled the _Synchronized Lyrics_ plugin at which point a similar effect (though half the visual size).

Further weight behind getting #152 sorted (which I'd started a while back) I guess. Can't find an issue about the plugin; am happy to either close this or leave it as tracking that problem..

**Comment by lazka** [comment]
#2050

**Comment by declension** [comment]
Hmm, different plugin and different effect but if it's the same root cause makes sense.

**Comment by lazka** [comment]
Ah, right, that's a different plugin. Not sure then. I can reproduce with the "Synchronized Lyrics" one

**Comment by lazka** [comment]
I've tried to fix the css hackery, but I'm not sure how it's supposed to look form the code. If it was invisible before then maybe newer gtk+ added default padding/min-height to the text widget.

**Comment by declension** [comment]
Hmm thanks. Doesn't seem to have changed it for me, but it's hardly an urgent bug.
